README.md

```diff
@@ -46,6 +46,12 @@ The plugin behavior can be controlled with the following list of settings
 - `default_device_role_color` string (default FF0000), color assigned to the device role if it needs to be created.
 - `default_management_interface` string (default "PLACEHOLDER"), name of the management interface that will be created, if one can't be identified on the device.
 - `default_management_prefix_length` integer ( default 0), length of the prefix that will be used for the management IP address, if the IP can't be found.
+- `platform_map` (dictionary), mapping of an **auto-detected** Netmiko platform to the **slug** name of your Platform. The dictionary should be in the format:
+    ```python
+    {
+      <NetBox Slug>: <desired_NAPALM_DRIVER> 
```

Comment:
Reading the code, this doesn't seem correct to me. Should it be `<netmiko platform>: <desired NAPALM driver>`?

Reply:
For this it is the slug of the platform in the NetBox instance. Such that if an organization wanted to map `Custom OS` to a particular NAPALM driver, since `custom_os` would not be found. In case organizations are running special OS, or just have a name that doesn't match what the automation industry had adopted.